packages/fxa-auth-server/scripts/scan-stripe-metadata.ts

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Stripe } from 'stripe';
+
+import { setupProcessingTaskObjects } from '../lib/payments/processing-tasks-setup';
+
+export async function init() {
+  const { stripeHelper } = await setupProcessingTaskObjects(
+    'scan-stripe-metadata'
+  );
+  const stripe: Stripe = stripeHelper.stripe;
+
+  const prices = [];
+  for await (const price of stripe.prices.list({
+    limit: 100,
+    product: 'prod_FvnsFHIfezy3ZI',
+    active: true,
+  })) {
+    prices.push({
+      name: price.nickname,
+      priceId: price.id,
+      interval: price.recurring?.interval,
+      productSet: price.metadata.productSet,
+      productOrder: price.metadata.productOrder,
+      // productSetOrder: price.metadata.productSetOrder,
```

Comment:
Remove?

Reply:
Actually didn't mean to include this file at all.... got hasty with a WIP commit. ;)